src/pretix/control/forms/item.py

```diff
@@ -207,14 +207,20 @@ class Meta:
 
 
 class QuotaForm(I18nModelForm):
+    itemvars = forms.MultipleChoiceField(
+        label=_("Products"),
+        required=True,
+    )
+
     def __init__(self, **kwargs):
         self.instance = kwargs.get('instance', None)
         self.event = kwargs.get('event')
         items = kwargs.pop('items', None) or self.event.items.prefetch_related('variations')
+        multiselect = kwargs.pop('multiselect', None)
```

Comment:
I don't feel "multiselect" really explains what this does. We always allow multiple selections, don't we? But I have trouble coming up with a better name. Maybe `searchable_selection` since that's the primary positive thing select2 gives us?

Reply:
Naming things is always difficult, isn't it? I also cannot think of anything better, so `searchable-selection` it is